test/integration/tests/button/props.js

```diff
@@ -17,6 +17,112 @@ describe(`paypal button component props`, () => {
         destroyTestContainer();
     });
 
+    it('should not render a paylater button when the experiment disables it', () => {
```

Comment:
Thanks for adding tests! Could you move them from props.js over to eligibility.js?

https://github.com/paypal/paypal-checkout-components/blob/master/test/integration/tests/button/eligibility.js